src/centerpoint/houston.py

```diff
@@ -77,6 +77,20 @@ def pennywise_conserve_saver6(x):
   if x <= 1000:
     return base + (.03+.0357)*500 + (.0357)*(x-500)
   return base + (.03+.0357)*500 + .0357*500 + (.119+.0357)*(x-1000)
+
+def texans_big_savings_plan(x):
+  base = 99
+  if x <= 999:
+    return base
+  if x <= 1500:
+    return base + (.038)*(x - 999)
+  if x > 1500:
+    return base + (.038)*500 + (.109)*(x - 1500)
+  
+def infuse_keep_it_simple_savings_3(x):
+  if x <= 1000:
+    return 39
```

Comment:
I think this should be 38
![image](https://user-images.githubusercontent.com/3519524/31322016-8e94ed96-ac54-11e7-9553-95ba93d8b2bf.png)